tests/sentry/models/tests.py

```diff
@@ -119,7 +119,6 @@ def test_does_transition_data_to_node(self):
         event = Event.objects.get(id=event_id)
         assert type(event.data) == NodeData
         assert event.data == data
-        assert event.data.id is None
```

Comment:
We now create an id for this record when it is created (or at least when it is accessed)